lib/utils/ConfigParser.js

```diff
@@ -98,7 +98,7 @@ class ConfigParser {
             /**
              * clone the original config
              */
-            var fileConfig = merge(require(filePath).config, {})
+            var fileConfig = merge(require(filePath).config, {}, { clone: false })
```

Comment:
Instead of passing the same parameters into the merge call everytime, how about making a constant like `MERGE_OPTIONS` and just pass these in?